<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldSettingsBox</class>
 <widget class="QDockWidget" name="WorldSettingsBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../_resources/editor.qrc">
    <normaloff>:/images/section.png</normaloff>:/images/section.png</iconset>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>World Settings</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents_3">
   <layout class="QGridLayout" name="gridLayout_18" rowstretch="0,0,0,0,0,0,0,0,0,0,80" columnstretch="20,80,0">
    <item row="9" column="0" colspan="3">
     <widget class="QTextEdit" name="WLD_Credirs">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoNone</set>
      </property>
      <property name="html">
       <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="WLD_DoCountStars">
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="label_58">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Intro level:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QGroupBox" name="WLD_DisableCharacters">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="title">
       <string>Disable characters:</string>
      </property>
      <layout class="QFormLayout" name="DisableCharactersL">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="horizontalSpacing">
        <number>2</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="WLD_AutostartLvl"/>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="WLD_Stars">
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QLabel" name="label_60">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Credits of this world (SMBX allows only 5 lines):</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_57">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>World title:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QCheckBox" name="WLD_RestartLevel">
      <property name="text">
       <string>Restart last level after fail</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QCheckBox" name="WLD_NoWorldMap">
      <property name="text">
       <string>Hub-styled world (without world map)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_59">
      <property name="text">
       <string>Total stars:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLineEdit" name="WLD_Title"/>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="Line" name="line_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="WLD_AutostartLvlBrowse">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../_resources/editor.qrc"/>
 </resources>
 <connections/>
</ui>
